lua/nvim-web-devicons.lua

```diff
@@ -91,9 +116,9 @@ local filetypes = {
   ["checkhealth"] = "checkhealth",
   ["commit"] = "commit_editmsg",
   ["copying"] = "copying",
-  ["gemfile"] = "gemfile$",
+  ["gemfile"] = "gemfile",
   ["lesser"] = "copying.lesser",
-  ["vagrantfile"] = "vagrantfile$",
+  ["vagrantfile"] = "vagrantfile",
```

Comment:
The `$` had no purpose and would not match. There are no more.

Reply:
I think that some programs use it for backup purposes, similar with `~` character

Reply:
Oh I see. I thought it was a regex.

I'll back them out.